v1-scripts/pull-from-upstream

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+UPSTREAM_REPOSITORY="${UPSTREAM_REPOSITORY:-https://github.com/MarlinFirmware/Marlin.git}"
+UPSTREAM_REMOTE_NAME="${UPSTREAM_REMOTE_NAME:-upstream}"
+UPSTREAM_BRANCH="${UPSTREAM_BRANCH:-$UPSTREAM_REMOTE_NAME/bugfix-2.0.x}"
+MIRROR_BRANCH="${MIRROR_BRANCH:-$UPSTREAM_BRANCH}"
+ORIGIN_REMOTE_NAME="${ORIGIN_REMOTE_NAME:-origin}"
+
+PULL_REQUESTS="${PULL_REQUESTS:-16108 16229}"
```

Comment:
Interesting.

Reply:
These have been merged, so we can get rid of these, right?

Reply:
This will also only work for PRs against bugfix, otherwise, it will merge stuff not in the PR.

Reply:
Yeah, these are not needed any more. I keep these around to test the concept.